infrastructure/template.yaml

```diff
@@ -1853,19 +1853,16 @@ Resources:
       AlarmActions:
         - !ImportValue platform-alarm-warning-alert-topic
       InsufficientDataActions: []
-      AlarmDescription: !Sub "${CheckHmrcEventBus} PutEvents (PutEvents Failed) 4 or more requests in the last hour. ${SupportManualURL}"
+      AlarmDescription: !Sub "${AWS::StackName}-${Environment} PutEvents (PutEvents Failed) 4 or more requests in the last hour. ${SupportManualURL}"
```

Comment:
If we are making changes to Alarms and descriptions we should remember to make changes here as https://team-manual.account.gov.uk/teams/CRI-Orange-team/supporting-cri-orange/hmrc-nino-runbook/#backend-api-alarms

Reply:
All the descriptions no longer tally with `4 or more requests in the last hour`

Reply:
Yep I'll raise a PR once/if these changes are approved.

Unsure what you mean with the last comment?